### ISSUE TITLE
Atom-Typescript error: The "path" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by BRandMol** [TypeStrong/atom-typescript#1580]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.55.0 x64
**Electron**: 6.1.12
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package 14.2.0

### Stack Trace

Atom-Typescript error: The "path" argument must be of type string. Received type undefined

```
At TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined

TypeError [ERR_INVALID_ARG_TYPE] [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:107:11)
    at Object.join (path.js:372:7)
    at PackageManager.resolvePackagePath (~/AppData/Local/atom/app-1.55.0/resources/app/static/<embedded>:11:375858)
    at /packages/atom-typescript/dist/main.js:9:1656)
    at /packages/atom-typescript/dist/main.js:9:8175
    at Array.map (<anonymous>)
    at /packages/atom-typescript/dist/main.js:9:8056)
    at /packages/atom-typescript/dist/main.js:9:288517
```

### Commands

```
     -1:43.4.0 find-and-replace:find-next (input.hidden-input)
  6x -1:38.5.0 core:undo (input.hidden-input)
  2x -1:27.1.0 find-and-replace:find-next (input.hidden-input)
     -0:20.7.0 find-and-replace:show (input.hidden-input)
```

### Non-Core Packages

```
angular-2-typeScript-snippets 0.7.0 
angular-snippets 10.0.3 
atom-bootstrap3 1.2.12 
atom-bootstrap4 1.4.0 
atom-typescript 14.2.0 
emmet 2.4.3 
file-icons 2.1.47 
minimap 4.39.10 
platformio-ide-terminal 2.10.1 
v-bootstrap4 0.2.2 
```

**Comment by nobodywasishere** [comment]
Also getting this issue.
Atom: 1.55.0 x64
Electron: 6.1.12
OS: Arch Linux 
Thrown from: atom-typescript 14.2.0

**Comment by lierdakil** [comment]
Thanks for reporting. This was a relatively benign oversight. Should be fixed in v14.2.1.

Side note: please read through the "installation" section in the README, specifically the "dependencies" part: https://github.com/TypeStrong/atom-typescript#installation
